tests/unit/json_test.c

```diff
@@ -1469,6 +1469,55 @@ static void test_string_escape(void)
         "Hello, world!\\n'Blah', \\\"blah\\\".");
 }
 
+#define assert_json5_data(_size, unescaped, escaped)              \
```

Comment:
Same here with a missing verb in the assert macro names.

Reply:
Alright, added.